app/data_migrations/populate_taxonomy.py

```diff
@@ -18,27 +18,40 @@ def populate_org_taxonomy(
 
     # First the org
     org = db.query(Organisation).filter(Organisation.name == org_name).one_or_none()
-    if org is None:
-        org = Organisation(
+
+    def add_org():
```

Comment:
This function needs to return the new `org.id` so you can use it to link in the taxonomy later.